tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.nuspec

```
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Datadog.Trace.Trimming</id>
+    <version>1.0.0</version>
+    <title>Datadog APM Trimmed Apps Support</title>
+    <authors>Datadog</authors>
+    <owners></owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <icon>packageIcon.png</icon>
+    <readme>README.md</readme>
+    <projectUrl>https://docs.datadoghq.com/tracing/trace_collection/dd_libraries</projectUrl>
+    <iconUrl>https://github.com/DataDog/dd-trace-dotnet/raw/master/datadog-logo-64x64.png</iconUrl>
+    <description>Datadog APM Trimmed Apps Support.</description>
```

Added some notes about automatic instrumentation. WDYT?

```suggestion
    <description>This package is required to enable Datadog automatic instrumentation on a trimmed app.</description>
```

---

added in [3d1cf41](https://github.com/DataDog/dd-trace-dotnet/pull/4195/commits/3d1cf41308db8b6afb6470ab9fdbca05697e10e1)